### PR TITLE
Joe transporation display

### PR DESCRIPTION
**Pull request by jcrotty**

### CODE REVIEW

File: public/views/transportation/transporation.controller.client.js [hunk 1]
```diff
@@ -11,32 +11,72 @@
...
+    var transportationId = $routeParams.transportationId;
     vm.backUrl = "#/trips/" + tripId;
 
     function init() {
-
+      TransportationService.findTransportationById(transportationId)
+        .success(function (transportation) {
+          vm.transportation = transportation;
+          vm.transportation.departureTime = new Date(transportation.departureTime);
+          vm.transportation.arrivalTime = new Date(transportation.arrivalTime);
```
Comment (alfred) [inline public/views/transportation/transporation.controller.client.js]: Why do you have to coerce these into `Date` when they are set to be them in the schema?

File: public/views/transportation/transporation.controller.client.js [hunk 2]
```diff
@@ -11,32 +11,72 @@
...
+          vm.error = "Error retrieving transportation.";
+          console.log("error:", err);
+        });
     }
     init();
+
+    function updateTransportation(newTransportation) {
+      TransportationService.updateTransportation(vm.transportation._id, newTransportation)
+        .success(function(updatedTransportation) {
+          vm.transportation = updatedTransportation;
```
Comment (alfred) [inline public/views/transportation/transporation.controller.client.js]: Why do you need to do this if you're navigating away from the page?

File: public/views/transportation/transporation.controller.client.js (hunk 1 above)
Comment (jcrotty) [inline public/views/transportation/transporation.controller.client.js]: the ng-model gives an error if you don't do this:  `Error: [ngModel:datefmt] Expected `1901-01-01T06:01:00.000Z` to be a date`

[StackOverflow](http://stackoverflow.com/questions/30537886/error-ngmodeldatefmt-expected-2015-05-29t190616-693209z-to-be-a-date-a)

**Comment by jcrotty** [comment]
A trip needs to have transportation to the trip and from the trip.  [origin](https://github.com/jcrotty/travel-dashboard/blob/joe-transporation-display/server/model/transportation/transportation.schema.server.js#L10) and [destination](https://github.com/jcrotty/travel-dashboard/blob/joe-transporation-display/server/model/transportation/transportation.schema.server.js#L23)

File: public/views/transportation/transporation.controller.client.js (hunk 2 above)
Comment (jcrotty) [inline public/views/transportation/transporation.controller.client.js]: I guess we don't have to

**Pull request merged**